### ISSUE TITLE
STRETCH GOAL: User (role of 'tenant') to edit parts of maintenance request

### ISSUE THREAD

**Issue by gregoryhaney** [gregoryhaney/nss-propczar#3]
NOTE: work this issue ONLY AFTER completing the following issue:
"STRETCH GOAL: Create tenant view of maint requests"
"[](https://github.com/gregoryhaney/nss-propczar/issues/22)"

Need to allow a user with role of "tenant" to make edits to maintenance request with the following restrictions:
1. request must be in 'open' status (`openStatus === false`)
2. request must have been submitted by the user 
3. update only the fields: `title` and `description`

**Comment by gregoryhaney** [comment]
Set functionality in `MaintRequestsList.js` module/component:

- user with role of "tenant" can only see maintenance requests for their property.
- if the status of the ticket is "closed" and the user clicks the EDIT button, they receive a window alert stating they cannot edit a closed ticket
- the DELETE button is not displayed for the user with a role of "tenant"
- new route `/maintRequests` added to `NavBar.js` for the tenant user for viewing all their maintenance request tickets
